content/homomorphisms/en/homomorphisms.md

```diff
@@ -129,7 +129,7 @@ b^{a_j}=\underbrace{b\cdot b\cdot\dots\cdot b}_{{a_j} \text{ times}}
 $$
 
 $$
-b^{a_i +a_j}=\underbrace{b\cdot b\cdot\dots\cdot b}_{{a_i} \text{ times}}\cdot\underbrace{b\cdot b\cdot\dots\cdot b}_{{a_j} \text{ times}}
+b^{a_i + a_j}=\underbrace{b\cdot b\cdot\dots\cdot b}_{{a_i} \text{ times}} \cdot \underbrace{b \cdot b \cdot \dots \cdot b} _{a_j \text{ times}}
```

Comment:
can we undo the changes in this file? It's not clear how this changes the formula rendering.